db/migrate/20211104171214_create_cap_usage_update_payload.rb

```diff
@@ -0,0 +1,14 @@
+class CreateCapUsageUpdatePayload < ActiveRecord::Migration[6.1]
+  def change
+    create_table :cap_usage_update_payloads do |t|
+      t.string :payload_id
+      t.string :payload_xml
```

Comment:
t.text?

Reply:
I checked that, it makes no difference...

https://www.postgresql.org/docs/9.1/datatype-character.html#:~:text=character%20without%20length%20specifier%20is,stores%20strings%20of%20any%20length.